ONSdigital/response-operations-ui
# RAS-1088 Implement 48 hour grace period for survey change of status

# What and why?
Updating confirm button to make it disabled when no radios can be selected
# How to test?
deploy this to your env, complete a survey and try to reverted to not started, the radio and the confirm button should be disabled
# Jira
[RAS-1088](https://jira.ons.gov.uk/browse/RAS-1088)
